syntax = "proto3";

enum Operation {
  UNKNOWN=0;
  PUT=1;
  DELETE=2;
  CREATE=3;
}



message order {
  int32 id = 1;
  int32 product_id = 2;
  string product_name = 3;
  int32 user_id = 4;
  int32 quantity = 5;
  float total_price = 6;
  Operation type = 7;
}